lib/active_merchant/billing/gateways/rapyd.rb

```diff
@@ -333,6 +334,10 @@ def commit(method, action, parameters)
           test: test?,
           error_code: error_code_from(response)
         )
+      rescue ActiveMerchant::ResponseError => e
+        response = e.response.body.present? ? parse(e.response.body) : { 'status' => { 'response_code' => e.response.msg } }
```

Comment:
Great, I think that this make sense with my [changes](https://github.com/activemerchant/active_merchant/pull/5018/files#diff-a7264544d772987646858e2b4287eb47f5e18340352a1d1ca4e2ad80c77a47c9R408).